service/lib/dinstaller/network.rb

```diff
@@ -31,24 +33,56 @@ def initialize(logger)
       @logger = logger
     end
 
-    # Probes the network configuration
-    def probe
-      logger.info "Probing network"
-      Yast::Lan.read_config
-      settings = Y2Network::ProposalSettings.instance
-      settings.apply_defaults
-      # force NetworkManager as we are not supporting other backends
-      settings.enable_network_manager!
-    end
-
     # Writes the network configuration to the installed system
+    #
+    # * Copies the connections configuration for NetworkManager, as D-Installer is not
+    #   performing further configuration of the network.
+    # * Enables the NetworkManager service.
     def install
-      Yast::WFM.CallFunction("save_network", [])
+      copy_files
```

Comment:
We could use the new config copier class instead of duplicating the logic, but in case of NM it is bare minimum..

https://github.com/yast/yast-network/pull/1318/files#diff-f65870082b39875af4467bb05dd5ae84681d5c2688c5e213257dc311b64eb4b2

And the new code needs to be reviewed yet.

Reply:
`
      Y2Network::ConfigCopier.for(:network_manager)&.copy
`